app/components/lib/socialShareBtn.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useRef, useCallback } from 'react'
+import injectSheet from 'react-jss'
+import { TwitterSVG } from './socialShareTwitterSVG'
+import { FacebookSVG } from './socialShareFacebookSVG'
+import { SocialShareSVG } from './socialShareSVG'
+import { useOnClickOutside } from '../hooks'
```

Comment:
why use relative path instead of direct?

Reply:
Because I don't think we have a jsconfig.json file

Reply:
either way trying to follow convention

Reply:
I'm inclined to accept code that works (and solves the issue) and create new issues in the backlog for improvements because we get it in front of users sooner for feedback, because there's always feedback on new features.  In this case, what I think @ZacharyLasky is talking about is that files like app/components/lib/socialShareFacebookSVG.jsx have svg that probably came from somewhere and it would be better to link to or pull from there, so that when that changes, we change, but it's a challenge to get SVG and include it in a react component.